src/EditorFeatures/CSharpTest/UseLocalFunction/UseLocalFunctionTests.cs

```diff
@@ -1932,9 +1932,7 @@ class C
     void M()
     {
         Action [||]local = null;
-        local = () => {
-            local.Invoke();
-        }
+        local = () => { local.Invoke(); }
```

Comment:
just making use of the fix... I wanted this to be on one line originally but didn't do that because of the bug